spec/index.html

```diff
@@ -17,7 +18,7 @@
     <script src="/testem.js"></script>
 
     <!-- include source files here... -->
-    <script src="/lib/annotator-custom-editor.js"></script>
+    <script src="/.tmp/annotator-custom-editor.js"></script>
```

Comment:
on reflection i should probably remove stuff the `.tmp` dir from source control.